examples/worlds/triggered_publisher.sdf

```diff
@@ -6,8 +6,7 @@ Send the command:
   ign topic -t "/start" -m ignition.msgs.Empty -p " "
 
 The vehicle should start moving while the two boxes remain floating. When the
-vehicle crosses the line on the ground, the first box should start falling. The
-moment the box hits the ground, the second box should start falling.
+vehicle crosses the line on the ground, the first box should start falling. Five simulation seconds after the box hits the ground, the second box should start falling.
```

Comment:
`One simulation second` instead of `Five`?

Reply:
Forgot to push this change, which makes it 5 seconds: 0299704fe72bb03f28de76790ce822a26258f4e5.